scripts/fortran_tools/fortran_write.py

```diff
@@ -210,7 +210,7 @@ def write(self, statement, indent_level, continue_line=False):
                     # This is probably a bad situation so we have to break
                     #   in an ugly spot
                     best = self.__line_max - 1
-                    if len(outstr) > best:
+                    if len(outstr) > best + 1:
```

Comment:
Does this work if `self.__line_max == 132` (Fortran maximum)?

Reply:
```
            variable_list(3) =                                                                    &
                 'tendency_of_dry_air_enthalpy_at_constant_pressure_due_to_shortwave_radiation_adjusted_by_air_pressure_thicknesse&
&s'
```
Is that not okay? I thought that is how you were supposed to split a character context across lines.

Reply:
oh! well, then, I suppose we're good on that front. i'm showing my fortran ignorance! so it's just the first issue that i'll look into a bit more tomorrow.

Reply:
Thanks again @gold2718

I took another stab at fixing the issue where the line is exactly equal to the max line length and doesn't need to be continued. If you have other ideas on how to fix it, I'm all ears!